core/project_settings.cpp

```diff
@@ -1047,6 +1047,10 @@ ProjectSettings::ProjectSettings() {
 	custom_prop_info["compression/formats/gzip/compression_level"] = PropertyInfo(Variant::INT, "compression/formats/gzip/compression_level", PROPERTY_HINT_RANGE, "-1,9,1");
 
 	using_datapack = false;
+
+    // Physics settings
+    Compression::zstd_level = GLOBAL_DEF("physics/3d/physics_engine", 0);
```

Comment:
The `Compression::zstd_level = ` is buggy, you should just start directly with the `GLOBAL_DEF()` without assigning it.

Also your indentation is wrong, it should use tabs.